common-arcana.lic

```diff
@@ -236,4 +236,57 @@ module DRCA
       return parse_mana_message(mana_msg)
     end
   end
+
+  def check_osrel(osrel_no_harness, osrel_amount)
+    return unless DRSpells.active_spells['Osrel Meraud'] && DRSpells.active_spells['Osrel Meraud'] < 90
+    infuse_om(!osrel_no_harness, osrel_amount)
+  end
+
+  def do_waggle_set(settings, spells)
```

Comment:
There's no reason this has to be called from waggle (or buff...). Maybe `cast_spells`?